docs/guides/docs/running-a-node/running-a-testnet-node.mdx

```diff
@@ -44,6 +38,18 @@ parent:
   trim="true"
 />
 
+<ConditionalContent versionSet={props.versionSet} showForVersions={["default"]}>
```

Comment:
Can you remove the conditional content component so this shows for default and nightly?